.github/workflows/release_process.yml

```diff
@@ -11,6 +11,26 @@ on:
         type: boolean
         required: false
         default: false
+      release_wamr_sdk:
+        description: "If the WAMR SDK release is required"
+        type: boolean
+        required: false
+        default: false
+      release_wamr_lldb:
+        description: "If the WAMR LLDB release is required"
+        type: boolean
+        required: false
+        default: false
+      release_wamr_ide_vscode_ext:
+        description: "If the WAMR VSCode extension release is required"
+        type: boolean
+        required: false
+        default: false
+      release_wamr_wasi_ext:
+        description: "If the WAMR WASI extensions release is required"
+        type: boolean
+        required: false
+        default: true
```

Comment:
@yamt please let me know if wamr-wasi-ext should be released by default?

Reply:
as it's cheap to build, i guess there is little reason to exclude it.